src/config.c

```diff
@@ -3236,6 +3236,7 @@ standardConfig static_configs[] = {
     createSizeTConfig("hll-sparse-max-bytes", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.hll_sparse_max_bytes, 3000, MEMORY_CONFIG, NULL, NULL),
     createSizeTConfig("tracking-table-max-keys", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.tracking_table_max_keys, 1000000, INTEGER_CONFIG, NULL, NULL), /* Default: 1 million keys max. */
     createSizeTConfig("client-query-buffer-limit", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, 1024*1024, LONG_MAX, server.client_max_querybuf_len, 1024*1024*1024, MEMORY_CONFIG, NULL, NULL), /* Default: 1GB max query buffer. */
+    createSizeTConfig("client-multi-buffer-limit", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, 1024*1024, LONG_MAX, server.client_multi_buffer_limit, 1024*1024*1024, MEMORY_CONFIG, NULL, NULL), /* Default: 1GB max multi memory used by buffered commands. */
```

Comment:
you didn't update redis.conf

Reply:
i'm not certain i like this config, and btw, both client output buffer limits and client eviction limit are by default 0, i suppose because we can't find a default number that's right for enough cases.

Reply:
the default value of `client-output-buffer-limit` is 1GB, so currently I am also setting the same default value for `client-multi-buffer-limit`.

Reply:
no, it's unlimited:
```c
/* Output buffer limits presets. */
clientBufferLimitsConfig clientBufferLimitsDefaults[CLIENT_TYPE_OBUF_COUNT] = {
    {0, 0, 0}, /* normal */
    {1024*1024*256, 1024*1024*64, 60}, /* slave */
    {1024*1024*32, 1024*1024*8, 60}  /* pubsub */
};
```

Reply:
Oops... I was mislead. I mean `client-query-buffer-limit` and its default value is 1GB, since the multi buffer should belong to input type.

```c
    createSizeTConfig("client-query-buffer-limit", NULL, DEBUG_CONFIG | MODIFIABLE_CONFIG, 1024*1024, LONG_MAX, server.client_max_querybuf_len, 1024*1024*1024, MEMORY_CONFIG, NULL, NULL),
    /* Default: 1GB max query buffer. */
```